token-metadata/program/src/utils/mod.rs

```diff
@@ -190,18 +192,42 @@ pub fn zero_account(s: &str, size: usize) -> String {
 pub fn close_program_account<'a>(
```

Comment:
Maybe we should have this as `pub(crate)` to avoid it being used outside token metadata – it appears on the suggestion list if you have token metadata as dependency.